product/views.py

```diff
@@ -51,5 +52,48 @@ def get(self, request, product_id):
         except KeyError:
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
         
+        except Product.DoesNotExist:
+            return JsonResponse({'message':'PRODUCT_DOES_NOT_EXIST'}, status=400)
+
+class ReviewLikeView(View):
+    # 상품 리뷰에 도움이 됐다를 표시하면 like 표시
+    def post(self, request, product_id):
+        try:
+            data     = json.loads(request.body)
+            review_id = data.get('review_id')
+
+            # user는 login_decorator로 불러온다.
+            email    = data.get('email')
+            password = data.get('password')
+
+            if not email:
+                return JsonResponse({'message':'NO_EMAIL_ERROR'}, status=400)
+
+            if not User.objects.filter(email=email).exists():
+                return JsonResponse({'message':'INVALID_USER'}, status=401)
+
+            user = User.objects.get(email=email)
+
+            if ProductReview.objects.filter(id=review_id).exists():
+                product_review = ProductReview.objects.get(id=review_id)
+            else:
+                return JsonResponse({'message':'INVALID_REVIEW'}, status=401)
+            
```

Comment:
```suggestion
            if not ProductReview.objects.filter(id=review_id).exists():
                return JsonResponse({'message':'INVALID_REVIEW'}, status=401)

            product_review = ProductReview.objects.get(id=review_id)
```
조건이 충족되지 않았을 때에 함수가 바로 return 할 수 있도록 하시면 더 좋을 것 같아요 👍

Reply:
네 수정하겠습니다!